daemon/cmd/daemon_main.go

```diff
@@ -1132,6 +1132,10 @@ func InitGlobalFlags(cmd *cobra.Command, vp *viper.Viper) {
 	flags.MarkHidden(option.EnableK8sNetworkPolicy)
 	option.BindEnv(vp, option.EnableK8sNetworkPolicy)
 
+	flags.Bool(option.EnableCiliumNetworkPolicy, defaults.EnableCiliumNetworkPolicy, "Enable support for Cilium NetworkPolicy")
```

Comment:
I don't think you merged the changes to update the flag/comments to clarify this enables both CNP and CCNP?

Reply:
Done.